meteor/client/ui/Status/package-status/index.tsx

```diff
@@ -13,131 +12,174 @@ import { doUserAction, UserAction } from '../../../lib/userAction'
 import { Studios } from '../../../../lib/collections/Studios'
 import { Meteor } from 'meteor/meteor'
 import { PackageStatus } from './PackageStatus'
-
-interface IIExpectedPackagesStatusTrackedProps {
-	expectedPackageWorkStatuses: ExpectedPackageWorkStatus[]
-	expectedPackages: ExpectedPackageDB[]
-}
-
-export const ExpectedPackagesStatus = translateWithTracker<{}, {}, IIExpectedPackagesStatusTrackedProps>(() => {
-	return {
-		expectedPackageWorkStatuses: ExpectedPackageWorkStatuses.find({}).fetch(),
-		expectedPackages: ExpectedPackages.find({}).fetch(),
+import { PackageContainerStatuses } from '../../../../lib/collections/PackageContainerStatus'
+import { PackageContainerStatus } from './PackageContainerStatus'
+import { Spinner } from '../../../lib/Spinner'
+import { useTranslation } from 'react-i18next'
+
+export const ExpectedPackagesStatus: React.FC<{}> = function ExpectedPackagesStatus(_props: {}) {
+	const { t } = useTranslation()
+
+	const studioIds = useTracker(
+		() =>
+			Studios.find()
+				.fetch()
+				.map((studio) => studio._id),
+		[]
+	)
+
+	const allSubsReady: boolean =
+		[
+			useSubscription(PubSub.expectedPackageWorkStatuses, {
+				studioId: { $in: studioIds },
+			}),
+			useSubscription(PubSub.expectedPackages, {
+				studioId: { $in: studioIds },
+			}),
+			useSubscription(PubSub.packageContainerStatuses, {
+				studioId: { $in: studioIds },
+			}),
+			studioIds && studioIds.length > 0,
+		].reduce((memo, value) => memo && value, true) || false
+
+	const expectedPackageWorkStatuses = useTracker(() => ExpectedPackageWorkStatuses.find({}).fetch(), []) || []
```

Comment:
Actually, useTracker has a third, optional, `initial` parameter you should use here. This is then the value returned if tracker hasn't returned anything yet.